.github/workflows/maven.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
         key: ${{ matrix.os }}-m2-${{ hashFiles('**/pom.xml') }}
         restore-keys: ${{ matrix.os }}-m2
     - name: Build with Maven
-      run: mvn -B -V -e "-Dstyle.color=always" verify
+      run: mvn -B -V -e "-Dstyle.color=always" verify -D"license.skip=true"
```

Comment:
I think it's really important that CI check to verify files are properly licensed. There probably wouldn't be errors if we dropped `license-maven-plugin-git`, since as far as I can tell, that's only used to populate a few optional copyright fields that might be used inside the license header template... but we have no reference to any of those variables in our license header template.